src/rasterizer/rasterizer.py

```diff
@@ -171,10 +171,12 @@ def __init__(self, resolution=256, steps=128, sigma=1e-2, method='base'):
         self.sigma = sigma
         self.method = method
 
-        self.gpu = torch.empty(2)
+        # TODO the follow line is required for the `tiled` method
+        # self.gpu = torch.empty(2)
         XX, YY = np.meshgrid(range(self.resolution), range(self.resolution))
-        XX_expanded = XX[np.newaxis, :, :]
-        YY_expanded = YY[np.newaxis, :, :]
+        YY = np.flip(YY)
+        XX_expanded = XX[:, :, np.newaxis]
+        YY_expanded = YY[:, :, np.newaxis]
```

Comment:
@ArianaFreitag this means that when in the `_raster_*` methods, we don't need to futz around with

`torch.transpose(torch.expand(self.x_meshgrid, [...]), ...)`

Instead, it's just

`torch.expand(self.x_meshgrid, [...])`